packages/shopify-api/lib/__tests__/test-config.ts

```diff
@@ -52,9 +52,11 @@ type TestConfig<Overrides extends TestOverridesOption<Future>, Future> = Modify<
  * This way, we'll always ensure our tests are covering all future flags. Please make sure to also have tests for the
  * old behaviour.
  */
-const TEST_FUTURE_FLAGS: Required<{[key in keyof FutureFlags]: true}> = {
+
+const TEST_FUTURE_FLAGS: Required<{
+  [key in keyof Omit<FutureFlags, 'unstable_tokenExchange'>]: true;
```

Comment:
Yeah I kept it because it's still being passed into the `future` object by the Remix package (and other consumers of the lib) and would lead to type errors:
<img width="994" alt="image" src="https://github.com/Shopify/shopify-api-js/assets/60748788/1e6ca17a-4e35-4229-a699-91ce25effb84">